### PR TITLE
Plasmamen Fire Sprite Bug

### LINKED ISSUE #18196: Plasmamen lose their special flames when cloned.
#### Description of issue
Normally plasmaman have a unique on fire sprite, but when you get cloned you lose it.

#### Difference between expected and actual behavior
Retaining my cool flames after being cloned instead of getting the normal ones

#### Steps to reproduce 
All it takes is being cloned.

#### Specific information for locating
Something that might be related i'm not sure, is health analyzers show roundstart plasmaman as 100% blood, but when you get cloned and have the normal flames they register you as proper 0% blood.

#### Length of time in which bug has been known to occur

I'm sure it's been around a while, i'm surprised I never noticed I had the normal flames while being dragged to cryo tube sooner.

#### Server revision

Revision: f5b9e6985f4586543d472af4737385dc0549c929

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by PublicEnemyMaiPai**
fixes #18196
:cl:
 * bugfix: Plasmamen now receive their old flaming sprite after cloning again.

### CODE REVIEW

**Comment by PublicEnemyMaiPai** [comment]
oh great what did i do wrong today?

File: code/modules/medical/cloning.dm [hunk 1]
```diff
@@ -327,6 +327,8 @@
 			//So clones don't die of oxyloss in a running pod.
 			else if(occupant.reagents.get_reagent_amount(INAPROVALINE) < 30) //Done like this because inaprovaline is toxic to vox
 				occupant.reagents.add_reagent(INAPROVALINE, 60)
+			if(isplasmaman(H))
```
Comment (MadmanMartian) [inline code/modules/medical/cloning.dm]: I didn't notice before, but this is in the wrong place.

Put this in the growClone proc so it only sets it once.

**Review by MadmanMartian — CHANGES_REQUESTED** [review]

File: code/modules/medical/cloning.dm (hunk 1 above)
Comment (PublicEnemyMaiPai) [reply code/modules/medical/cloning.dm]: alrighty my bad, ill do this tomorrow

File: code/modules/medical/cloning.dm [hunk 2]
```diff
@@ -216,6 +216,8 @@
 					if((G.mind && (G.mind.current.stat != DEAD) ||  G.mind != clonemind))
 						return FALSE
 
+					if(isplasmaman(H))
```
Comment (MadmanMartian) [inline code/modules/medical/cloning.dm]: There isn't a human reference at this point. Put it somewhere below line 230

**Review by MadmanMartian — CHANGES_REQUESTED** [review]

File: code/modules/medical/cloning.dm [hunk 3]
```diff
@@ -236,6 +235,9 @@
 
 	connected.update_icon()
 
+	if(isplasmaman(H))
+	H.fire_sprite = "Plasmaman"
```
Comment (MadmanMartian) [inline code/modules/medical/cloning.dm]: Gotta tab this one across so it's under the if conditional

**Review by MadmanMartian — APPROVED** [review]

**Pull request merged**